src/script/miniscript.h

```diff
@@ -1030,29 +1030,39 @@ inline NodeRef<Key> Parse(Span<const char> in, const Ctx& ctx)
 {
     using namespace spanparsing;
 
+    // Account for the number of fragments we started to parse. Any fragment is at least one byte
+    // long. If we started to parse more than 3600 of them, don't bother continuing: the resulting
+    // Miniscript would be invalid.
+    size_t opened_frags = 0;
+
     // The two integers are used to hold state for thresh()
     std::vector<std::tuple<ParseContext, int64_t, int64_t>> to_parse;
     std::vector<NodeRef<Key>> constructed;
 
     to_parse.emplace_back(ParseContext::WRAPPED_EXPR, -1, -1);
 
     while (!to_parse.empty()) {
+        if (opened_frags > MAX_STANDARD_P2WSH_SCRIPT_SIZE) return {};
```

Comment:
IIUC this limit is specific to Miniscript inside P2WSH. That's the case now for our descriptor implementation. Maybe use `MAX_SCRIPT_SIZE` (10,000) instead? 

See also Resource Limitations under https://bitcoin.sipa.be/miniscript/

Reply:
The Miniscript code already is P2WSH specifc. We can change this once we implement Tapscript support in Miniscript.